include/SFML/Graphics/PrimitiveType.hpp

```diff
@@ -40,11 +40,16 @@ enum PrimitiveType
 {
     Points,         ///< List of individual points
     Lines,          ///< List of individual lines
-    LinesStrip,     ///< List of connected lines, a point uses the previous point to form a line
+    LineStrip,      ///< List of connected lines, a point uses the previous point to form a line
     Triangles,      ///< List of individual triangles
-    TrianglesStrip, ///< List of connected triangles, a point uses the two previous points to form a triangle
-    TrianglesFan,   ///< List of connected triangles, a point uses the common center and the previous point to form a triangle
-    Quads           ///< List of individual quads (deprecated, don't work with OpenGL ES)
+    TriangleStrip,  ///< List of connected triangles, a point uses the two previous points to form a triangle
```

Comment:
```TrianglesStrip``` was the longest name in the enum, now that it lost a character the start of all the documentation should be moved back by a single space.